src/content/dao/index.md

```diff
@@ -79,25 +129,6 @@ _Typically used for decentralized development and governance of protocols and da
 
 [DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and manage funds, meaning no one can buy their way into influencing its future.
 
-## How do DAOs work? {#how-daos-work}
```

Comment:
Why we removing this section? (Has it been moved? I don't see elsewhere)

Reply:
I moved it higher up on the page, above the DAO membership section (this section is way more important than the DAO membership one)

<img width="595" alt="image" src="https://user-images.githubusercontent.com/1193692/186222034-e35dbe43-e2ab-4f35-8b64-7b31dd7faf83.png">